openedx/core/djangoapps/schedules/management/commands/send_recurring_nudge.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import print_function
+
+import datetime
+import logging
+import pytz
+from django.contrib.sites.models import Site
+
+from celery import task
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from django.core.urlresolvers import reverse
+
+from openedx.core.djangoapps.schedules.models import Schedule, ScheduleConfig
+
+from edx_ace.message import MessageType
+from edx_ace.recipient_resolver import RecipientResolver
+from edx_ace import ace
+from edx_ace.recipient import Recipient
+
+
+LOG = logging.getLogger(__name__)
+
+
+class RecurringNudge(MessageType):
+    def __init__(self, week, *args, **kwargs):
+        super(RecurringNudge, self).__init__(*args, **kwargs)
+        self.name = "recurringnudge_week{}".format(week)
```

Comment:
I was trying to keep this consistent with the standard template naming (which just lowercases the class name).